scripts/fits-image.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+
+import argparse
+
+import katsdpimageutils.render as render
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        description='Create image from FITS file written by katsdpimager.'
```

Comment:
Remove the 'katsdpimager' here since this script will work for FITS files from the continuum imager as well.

Reply:
Done - thanks for catching it.